SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -101,25 +69,12 @@ void USpatialPlayerSpawner::SendPlayerSpawnRequest()
 			FURL LoginURL;
```

Comment:
Lots of params, any sense in grouping them into a struct?

Reply:
I'll put you in a struct